.env.dist

```diff
@@ -0,0 +1,5 @@
+MYSQL_ROOT_PASSWORD=root
```

Comment:
why do you need this in bundle?
If it's for test purposes, define it in `phpunit.xml`

Reply:
In order to be able to run the tests in local environment - values should be used both in travis and docker-compose configurations.